src/test/kotlin/com/atlassian/performance/tools/awsinfrastructure/AwsCliIT.kt

```diff
@@ -0,0 +1,31 @@
+package com.atlassian.performance.tools.awsinfrastructure
+
+import com.atlassian.performance.tools.sshubuntu.api.SshUbuntuContainer
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import java.util.concurrent.Executors
+import java.util.concurrent.TimeUnit
+
+class AwsCliIT {
+
+    @Test
+    fun shouldBeParallelOnDifferentSshConnections() {
```

Comment:
This doesn't use `IntegrationTestRuntime`, so it doesn't get the logging config and ends up with this:
> ERROR StatusLogger No log4j2 configuration file found. Using default configuration: logging only errors to the console. Set system property 'log4j2.debug' to show Log4j2 internal initialization logging.

Reply:
I'm not aware of `IntegrationTestRuntime`. How should I use it? Is it documented somewhere?

Reply:
You don't have to use it. It's an internal "helper", so no wonder there's no documentation.
But this is the first IT that doesn't use, so only this IT has a log4j problem.
Gotta config log4j one way or another and that is well documented: https://logging.apache.org/log4j/2.x/manual/configuration.html